__tests__/app.test.js

```diff
@@ -48,3 +48,70 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("return status 200 when successful", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200);
+  });
+  test("return an object with the expected values", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .then(({ body: article }) => {
+        expect(article.article).toEqual(
+          expect.objectContaining({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number)
+          })
+        );
+      });
+  });
```

Comment:
You should add a test where you check you have retrieved article 1 when you request /api/articles/1
I.e. it should check for the specific values that article 1, i.e. you would expect the article id to equal 1.
The reason for this is, you could currently be returning any article.